pyproject.toml

```diff
@@ -36,7 +36,8 @@ Pillow = "*"
 pycryptodome = "*"
 PyJWT = "*"
 python = ">=3.8,<4"
-requests = "*"
+pyyaml = ">=6.0.2"
```

Comment:
Since we don't directly depend on `pyyaml`, i don't think it should go into the `pyproject.toml` file. If you need the version locked you should be able to drop it here and run `poetry lock --no-update` to update the hash. That will perserve the version you have in the lock file without adding a unnecessary dependency here.